src/pages/Navigator-Page-2.md

```diff
@@ -0,0 +1,39 @@
+Navigator Page 2:
+
+
+      <Navigator leftRailText='Home sweet home' stepprogress={ false }>
+        <GeneralForm
+          formName='ExampleForm'
+          onSubmit={ () => alert('FormSubmitted!') }
+        >
+          <TextComponent tag='p' type={ 4 } semibold>
+            Great, just a few more questions.
+          </TextComponent>
+          <ToggleField
+            htmlFor=''
```

Comment:
@cisacke Err... change of opinion. This isn't actually a form input. I think it was supposed to be a radio button implementation... but I don't think that's necessary since we're using React + Redux. I removed it.